### PR TITLE
Replace all inappropriate links to English pages in Japanese documentation.

### PR DESCRIPTION
**Pull request by riita10069**
Ref: #31140 

This PR was generated by a script.
Legitimacy is ensured by manual work.
At the same time, I aim to integrate this into the Prow Job.
Therefore, the following two issues need to be considered.

- The detected links are legitimate.
- There are no undetected inappropriate links.

:warning: I have not completed the check of all links by self review.
When my self review is complete, I will comment on it.

### CODE REVIEW

**Comment by riita10069** [comment]
/retitle Replace all inappropriate links to English pages in Japanese documentation.

**Comment by AvineshTripathi** [comment]
Hey @riita10069 if you can share the script it would be helpful for other localisation too and maybe can help with this [issue](https://github.com/kubernetes/contributor-site/issues/236l)

**Comment by riita10069** [comment]
@AvineshTripathi 

Thanks for your comment!

Is this what you want to say?
https://github.com/kubernetes/contributor-site/issues/236

I'll share the script soon.
Give me a minute.

**Comment by riita10069** [comment]
I've published the program I used to create this PR.
https://gist.github.com/riita10069/040509ff774a878becc8ab604cee9656

**Comment by AvineshTripathi** [comment]
Hey @riita10069 going through the code there were few things I wanted to ask 
- in place of doing recursion in tree() can we use WalkDir() to do same thing
- I think in the code uses to search for path in local directory, can we code something which makes this local independent.

Please correct me if I am wrong (Quite new with go)

**Comment by riita10069** [comment]
> in place of doing recursion in tree() can we use WalkDir() to do same thing

maybe that's correct!
I'll change it if I need to.

> I think in the code uses to search for path in local directory, can we code something which makes this local independent.

This script is written to be executed in `website/scripts` but, It goes beyond that, it is intended to be run in CI (i.e., a prow job).
If you could be more specific about the problem with this code, I would be happy to help.

**Comment by AvineshTripathi** [comment]
Thanks for replying I ll do a deep dive in prow job thing how exactly it work :+1:

File: content/ja/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 1]
```diff
@@ -241,4 +241,4 @@ spec:
     app: iis-2019
 ```
 
-[RuntimeClass]: https://kubernetes.io/docs/concepts/containers/runtime-class/
+[RuntimeClass]: https://kubernetes.io/ja/docs/concepts/containers/runtime-class/
```
Comment (riita10069) [inline content/ja/docs/setup/production-environment/windows/user-guide-windows-containers.md:244]: It is right that there is a Japanese link, but should this be defined as a relative path?

Comment (nasa9084) [reply content/ja/docs/setup/production-environment/windows/user-guide-windows-containers.md:244]: depends on base(en) document

**Pull request merged**